### ISSUE TITLE
Ability to define update groups (monorepos)

### ISSUE THREAD

**Issue by wolfy1339** [dependabot/feedback#543]
With Greenkeeper, one cool feature is that I can have it update a dependency across multiple packages in the same repo in a single PR.

In the config you would create these groups and they would be (example for JavaScript) an array containing paths to the package root relative to the repository's root

**Comment by mwarkentin** [comment]
We have some repos using Renovate and it seems to have similar capabilities - our devs would like to have the ability to group minor and major upgrades separately:

> I don't think it offers a convenient way to split out minor package upgrades from major ones (see "Groups" https://renovatebot.com/docs/presets-group/)

**Comment by gperdomor** [comment]
**Current:**
<img width="996" alt="Screen Shot 2019-09-24 at 4 51 54 PM" src="https://user-images.githubusercontent.com/371939/65545412-b2cd7180-deeb-11e9-9717-289dcbf7a7ec.png">

**Expected:**
<img width="1000" alt="Screen Shot 2019-09-24 at 4 52 07 PM" src="https://user-images.githubusercontent.com/371939/65545471-d0024000-deeb-11e9-95a3-0dec23858a57.png">

**Comment by wolfy1339** [comment]
I don't think I explained this well enough. What I want is an option so that dependabot only opens one PR which updates a given dependency across all the packages in a monorepo instead of one per package in that repo

**Comment by EvHaus** [comment]
Agreed. Dependabot is much better than greenkeeper in many ways, but this is something I miss after migrating away from greenkeeper very much. Having to merge separate PRs for what is ultimately a single update (eg. `babel`) causes all sorts of rebase conflicts, etc...

**Comment by rebelagentm** [comment]
👋 Supporting ways to group pull requests is high on our list of enhancements to add to Dependabot as soon as our team gets the capacity to focus on them.
